app/ide-desktop/lib/dashboard/src/index.tsx

```diff
@@ -0,0 +1,15 @@
+/** @file Index file declaring main DOM structure for the app. */
+
+import * as authentication from "enso-studio-authentication";
+
+import * as app from "./authentication/src/components/app";
```

Comment:
consider inlining `props` into the function call so that this import can be removed

Reply:
With the addition and re-export of `Platform` this is no longer possible.